dashboard/app/batch_coverage.go

```diff
@@ -151,3 +151,15 @@ func nsDataAvailable(ctx context.Context, ns string) ([]coveragedb.TimePeriod, [
 	}
 	return periods, recordsCount, nil
 }
+
+func handleBatchCoverageClean(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+	totalDeleted, err := coveragedb.DeleteGarbage(ctx)
+	if err != nil {
+		errMsg := fmt.Sprintf("failed to coveragedb.DeleteGarbage: %s", err.Error())
+		log.Errorf(ctx, "%s", errMsg)
+		w.Write([]byte(errMsg))
+		return
+	}
+	w.Write([]byte(fmt.Sprintf("deleted %d rows\n", totalDeleted)))
```

Comment:
Nit: I'd also log this figure to have some history in Cloud Logging.

Reply:
done